client/views/curatorInbox/sourceIncidentReportsHeader.jade

```diff
@@ -0,0 +1,22 @@
+template(name="sourceIncidentReportsHeader")
+  .curator-source-details--sub-header
+    button.btn.btn-sm.btn-danger.delete(
+      disabled="{{#unless incidentsSelectedCount}} true {{/unless}}")
+      if deleting
```

Comment:
Oh, nevermind...the local collection only controls what's selected.
I added it because there's a significant delay locally for me.